cloudify_aws/rds/resources/instance.py

```diff
@@ -110,7 +111,7 @@ def create(ctx, iface, resource_config, **_):
             utils.update_resource_arn(ctx.instance, value)
             continue
         elif isinstance(value, datetime):
-            value = str(value)
+            value = text_type(value)
```

Comment:
Same question. No change is necessary, just curious.

Reply:
Same as above